packages/backend/src/core/NoteCreateService.ts

```diff
@@ -313,7 +352,10 @@ export class NoteCreateService {
 
 		const note = await this.insertNote(user, data, tags, emojis, mentionedUsers);
 
-		setImmediate(() => this.postNoteCreated(note, user, data, silent, tags!, mentionedUsers!));
+		// FIXME
+		if (process.env.NODE_ENV !== 'test') {
+			setImmediate(() => this.postNoteCreated(note, user, data, silent, tags!, mentionedUsers!));
+		}
```

Comment:
The following error will be thrown if `setImmediate` is enabled in Vitest. What's the purpose of this `setImmediate` here in the first place? Might be a good idea to add a comment.

```
Vitest caught 3 unhandled errors during the test run.
This might cause false positive tests. Resolve unhandled errors to make sure your tests are not affected.

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯ Unhandled Rejection ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
TypeORMError: Driver not Connected
 ❯ PostgresDriver.obtainMasterConnection ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/driver/postgres/PostgresDriver.ts:1157:19
 ❯ PostgresQueryRunner.connect ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/driver/postgres/PostgresQueryRunner.ts:110:18
 ❯ PostgresQueryRunner.query ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/driver/postgres/PostgresQueryRunner.ts:244:47
 ❯ UpdateQueryBuilder.execute ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/query-builder/UpdateQueryBuilder.ts:145:51
 ❯ processTicksAndRejections node:internal/process/task_queues:95:5
 ❯ runNextTicks node:internal/process/task_queues:64:3
 ❯ processImmediate node:internal/timers:442:9

This error originated in "test/unit/activitypub.ts" test file. It doesn't mean the error was thrown inside the file itself, but while it was running.
The latest test that might've caused the error is "Actor". It might mean one of the following:
- The error was thrown, while Vitest was running this test.
- This was the last recorded test before the error was thrown, if error originated after test finished its execution.

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯ Unhandled Rejection ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
TypeORMError: Driver not Connected
 ❯ PostgresDriver.obtainMasterConnection ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/driver/postgres/PostgresDriver.ts:1157:19
 ❯ PostgresQueryRunner.connect ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/driver/postgres/PostgresQueryRunner.ts:110:18
 ❯ PostgresQueryRunner.query ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/driver/postgres/PostgresQueryRunner.ts:244:47
 ❯ SelectQueryBuilder.loadRawResults ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/query-builder/SelectQueryBuilder.ts:3694:43
 ❯ SelectQueryBuilder.executeEntitiesAndRawResults ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/query-builder/SelectQueryBuilder.ts:3459:37
 ❯ SelectQueryBuilder.getRawAndEntities ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/query-builder/SelectQueryBuilder.ts:1646:40
 ❯ SelectQueryBuilder.getMany ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/query-builder/SelectQueryBuilder.ts:1736:36
 ❯ EntityManager.find ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/entity-manager/EntityManager.ts:1015:14
 ❯ Repository.find ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/repository/Repository.ts:483:29
 ❯ src/core/NoteCreateService.ts:489:65
    487| 
    488|   // Word mute
    489|   mutedWordsCache.fetch(null, () => this.userProfilesRepository.find({
       |                                                                 ^
    490|    where: {
    491|     enableWordMute: true,

This error originated in "test/unit/activitypub.ts" test file. It doesn't mean the error was thrown inside the file itself, but while it was running.
The latest test that might've caused the error is "Actor". It might mean one of the following:
- The error was thrown, while Vitest was running this test.
- This was the last recorded test before the error was thrown, if error originated after test finished its execution.

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯ Unhandled Rejection ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
TypeORMError: Driver not Connected
 ❯ PostgresDriver.obtainMasterConnection ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/driver/postgres/PostgresDriver.ts:1157:19
 ❯ PostgresQueryRunner.connect ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/driver/postgres/PostgresQueryRunner.ts:110:18
 ❯ PostgresQueryRunner.query ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/driver/postgres/PostgresQueryRunner.ts:244:47
 ❯ SelectQueryBuilder.loadRawResults ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/query-builder/SelectQueryBuilder.ts:3694:43
 ❯ SelectQueryBuilder.executeEntitiesAndRawResults ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/query-builder/SelectQueryBuilder.ts:3459:37
 ❯ SelectQueryBuilder.getRawAndEntities ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/query-builder/SelectQueryBuilder.ts:1646:40
 ❯ SelectQueryBuilder.getMany ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/query-builder/SelectQueryBuilder.ts:1736:36
 ❯ EntityManager.find ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/entity-manager/EntityManager.ts:1015:14
 ❯ Repository.find ../../node_modules/.pnpm/typeorm@0.3.11_ioredis@4.28.5+pg@8.9.0/src/repository/Repository.ts:483:29
 ❯ AntennaService.getAntennas src/core/AntennaService.ts:227:50
    225|  public async getAntennas() {
    226|   if (!this.antennasFetched) {
    227|    this.antennas = await this.antennasRepository.find();
       |                                                  ^
    228|    this.antennasFetched = true;
    229|   }

This error originated in "test/unit/activitypub.ts" test file. It doesn't mean the error was thrown inside the file itself, but while it was running.
The latest test that might've caused the error is "Actor". It might mean one of the following:
- The error was thrown, while Vitest was running this test.
- This was the last recorded test before the error was thrown, if error originated after test finished its execution.
```